Interface.c

```diff
@@ -307,4 +307,69 @@ void executeRedirectCommand(char** args,int argsSize) {
 	
 }
 
+void executePipesCommand(char **args,int argsSize){
+	pid_t pid;
+	int fd[2],returnVal;;
+
+	if((pipe(fd)<0)){
+        	printf("Error creating pipe.\n");
+        	exit(EXIT_FAILURE);
+    	}
+
+	if((pid=fork())<0){
+        	printf("Error forking.\n");
+        	exit(EXIT_FAILURE);    
+    	}
+
+	char *firstCmd = args[0],*firstArg = args[1],*secondCmd = NULL,*secondArg = NULL;
```

Comment:
separate variable declarations or else it's hard to read. We should not be picky about ours line of code